crates/flux-syntax/src/lexer.rs

```diff
@@ -49,8 +49,14 @@ pub enum Token {
     Ensures,
     Literal(Lit),
     Ident(Symbol),
-    OpenDelim(Delimiter),
-    CloseDelim(Delimiter),
+    OpenParen,
+    CloseParen,
+    OpenBrace,
+    CloseBrace,
+    OpenBracket,
+    CloseBracket,
+    OpenInvisible(InvisibleOrigin),
+    CloseInvisible(InvisibleOrigin),
```

Comment:
I'm confused, why did you split this?

Reply:
because the rust people did so ... https://doc.rust-lang.org/nightly/nightly-rustc/rustc_ast/token/enum.TokenKind.html#variant.OpenParen ...

Reply:
No, it's fine like this